doc/user/content/sql/create-source/load-generator.md

```diff
@@ -53,7 +53,7 @@ interval, the next number in the sequence is emitted.
 
 The auction load generator simulates an auction house, where users are bidding
 on an ongoing series of auctions. The auction source is meant to be used
-with [`CREATE VIEWS`](/sql/create-views), which will create the following five
+with , which will create the following five
```

Comment:
missing word! I think you need `FOR ALL TABLES` here? This should be added to the load generator bnf as well.

Reply:
This is going to get fixed https://github.com/MaterializeInc/materialize/pull/15370 and I'm just going to overwrite this with that commit whenever it merges.